src/test/regress/expected/pg15_jsonpath.out

```diff
@@ -0,0 +1,354 @@
+ $ ? (@.a < -1e+1)                                            | $?(@."a" < -10)
+ $ ? (@.a < +1e+1)                                            | $?(@."a" < 10)
+ $ ? (@.a < .1e+1)                                            | $?(@."a" < 1)
+ $ ? (@.a < -.1e+1)                                           | $?(@."a" < -1)
+ $ ? (@.a < +.1e+1)                                           | $?(@."a" < 1)
+ $ ? (@.a < 0.1e+1)                                           | $?(@."a" < 1)
+ $ ? (@.a < -0.1e+1)                                          | $?(@."a" < -1)
+ $ ? (@.a < +0.1e+1)                                          | $?(@."a" < 1)
+ $ ? (@.a < 10.1e+1)                                          | $?(@."a" < 101)
+ $ ? (@.a < -10.1e+1)                                         | $?(@."a" < -101)
+ $ ? (@.a < +10.1e+1)                                         | $?(@."a" < 101)
+ 0                                                            | 0
+ 0.0                                                          | 0.0
+ 0.000                                                        | 0.000
+ 0.000e1                                                      | 0.00
+ 0.000e2                                                      | 0.0
+ 0.000e3                                                      | 0
+ 0.0010                                                       | 0.0010
+ 0.0010e-1                                                    | 0.00010
+ 0.0010e+1                                                    | 0.010
+ 0.0010e+2                                                    | 0.10
+ .001                                                         | 0.001
+ .001e1                                                       | 0.01
+ 1.                                                           | 1
+ 1.e1                                                         | 10
+ 1.2.e                                                        | (1.2)."e"
+ (1.2).e                                                      | (1.2)."e"
+ 1e3                                                          | 1000
+ 1.e3                                                         | 1000
+ 1.e3.e                                                       | (1000)."e"
+ 1.e3.e4                                                      | (1000)."e4"
+ 1.2e3                                                        | 1200
+ 1.2.e3                                                       | (1.2)."e3"
+ (1.2).e3                                                     | (1.2)."e3"
+ 1..e                                                         | (1)."e"
+ 1..e3                                                        | (1)."e3"
+ (1.).e                                                       | (1)."e"
+ (1.).e3                                                      | (1)."e3"
+ 1?(2>3)                                                      | (1)?(2 > 3)
+(158 rows)
+
+DROP SCHEMA jsonpath;
```

Comment:
Use `DROP SCHEMA jsonpath CASCADE;` here

Reply:
Good catch. Thanks. I missed the error messages at the end due to missing cascade.